src/components/DocumentWithEvaluations/index.tsx

```diff
@@ -15,6 +15,25 @@ export function DocumentWithEvaluations({
 }: DocumentWithReviewsProps) {
   const hasEvaluations = document.reviews && document.reviews.length > 0;
   
+  // Check for pending/running jobs across all evaluations
+  const pendingJobs = useMemo(() => {
+    return document.reviews.flatMap(review => 
+      review.jobs?.filter(job => job.status === 'PENDING' || job.status === 'RUNNING') || []
+    );
+  }, [document.reviews]);
```

Comment:
keep in mind we just want the recent jobs if they're the most recent job on each evaluation. So I imagine, on each evaluation, we'd query for "recentJob" or something like that.